redbot/core/events.py

```diff
@@ -241,10 +239,112 @@ async def on_command_error(ctx, error, unhandled_by_cog=False):
             await ctx.send(msg)
             if error.send_cmd_help:
                 await ctx.send_help()
+        elif isinstance(error, commands.RangeError):
+            if isinstance(error.value, int):
+                if (error.minimum == 0 or error.minimum == 1) and error.maximum is None:
```

Comment:
Whilst this check makes sense, if the minimum were to be 1, and "0" is passed to the converter, the message would still ask for a positive integer despite zero being a positive integer. Is the `== 1` check really needed?